src/EnergyPlus/AirflowNetwork/src/Elements.cpp

```diff
@@ -3690,6 +3681,1015 @@ namespace AirflowNetwork {
+                                            const Array1D<Real64> &BetaT,  // Density gradients in the TO zone (starting at linkheight) [Kg/m3/m]
+                                            const Array1D<Real64> &RhoStF, // Density at the start heights of Layers in the FROM zone
+                                            const Array1D<Real64> &RhoStT, // Density at the start heights of Layers in the TO zone
+                                            int const From,                // Number of FROM zone
+                                            int const To,                  // Number of To zone
+                                            Real64 const ActLh,            // Actual height of opening [m]
+                                            Real64 const OwnHeightFactor // Cosine of deviation angle of the opening plane from the vertical direction
+    )
+    {
+
+        // SUBROUTINE INFORMATION:
+        //       AUTHOR         Lixing Gu
+        //       DATE WRITTEN   Oct. 2005
+        //       MODIFIED       na
+        //       RE-ENGINEERED  This subroutine is revised based on PresProfile subroutine from COMIS
+
+        // PURPOSE OF THIS SUBROUTINE:
+        // This subroutine calculates for a large opening profiles of stack pressure difference and
+        // densities in the zones linked by the a detailed opening cmponent.
+
+        // METHODOLOGY EMPLOYED:
+        // The profiles are obtained in the following
+        // way:    - the opening is divided into NrInt vertical intervals
+        //         - the stack pressure difference and densities in From-
+        //           and To-zone are calculated at the centre of each
+        //           interval aswell as at the top and bottom of the LO
+        //          - these values are stored in the (NrInt+2)-dimensional
+        //             arrays DpProf, RhoProfF, RhoProfT.
+        // The calculation of stack pressure and density in the two zones
+        // is based on the arrays DpF/T, RhoStF/T, BetaF/T. These arrays
+        // are calculated in the COMIS routine Lclimb. They contain the
+        // values of stack pressure and density at the startheight of the
+        // opening and at startheights of all layers lying inside the
+        // opening, and the density gradients across the layers.
+        // The effective startheight zl(1/2) in the From/To zone and the
+        // effective length actLh of the LO take into account the
+        // startheightfactor, heightfactor and ownheightfactor. Thus for
+        // slanted windows the range of the profiles is the vertical
+        // projection of the actual opening.
+
+        // REFERENCES:
+        // Helmut E. Feustel and Alison Rayner-Hooson, "COMIS Fundamentals," LBL-28560,
+        // Lawrence Berkeley National Laboratory, Berkeley, CA, May 1990
+
+        // USE STATEMENTS:
```

Comment:
I believe we are eliminating empty comment sections. Use statements are evil anyways.

Reply:
Will do.